docs/cubeviz/displaycubes.rst

```diff
@@ -114,4 +114,80 @@ Specviz.
 Display Settings
 ================
 
-Coming soon
+To access all of the different display settings for an image viewer, click
+the :guilabel:`hammer and screwdriver` and go to the :guilabel:`Layer` tab.
+
+.. image:: img/display_settings.png
+
+Layer
+-----
+
+This option allows you to change which layer you are changing the settings for.
+
+Attribute
+---------
+
+This shows which extension of the data is being displayed in the current viewer.
+
+Contour
+-------
+
+This option selects whether to show or hide contours.
+
+Bitmap
+------
+
+This option selects whether to show or hide the bitmap (image) in the viewer.
+
+Opacity
+^^^^^^^
+
+Change the translucence of the image.
+
+Contrast
+^^^^^^^^
+
+Change the luminance of the color in the image.
+
+Bias
+^^^^
+
+Set a constant to subtract from every point in the data array before
+applying the conversion between data value and displayed pixel saturation.
+For example, with a data min=2 and max=10, any points in the array with value 2 or
+less would be displayed as black in the viewer and any points 10 or higher
+as white (with gray in between, assuming a gray scale colormap).
+The :guilabel:`bias` slider center position is 0 bias, such that you can apply negative
+bias values by sliding it left.
+
+Stretch
+^^^^^^^
+
+Change the equation that is used to convert data values between
+:guilabel:`min` and :guilabel:`max` to the 0 to 1 scale of pixel saturation on the displayed
+image.
+
+Percentile
+^^^^^^^^^^
+
+Change the percentage of data that is included in the visualization.
+
+Min
+"""
+
+If the percentile is "custom", then the minimum value for data to be visualized can be set.
```

Comment:
```suggestion
If the percentile is "custom", then the data value corresponding to the minimum of the colormap scale (e.g. black in grayscale) can be set.
```
After thinking about it more, and in light of the bias/stretch changes, I think this would be clearer.